apps/web/components/team/MemberList.tsx

```diff
@@ -4,7 +4,7 @@ import MemberListItem from "./MemberListItem";
 
 interface Props {
   team: inferQueryOutput<"viewer.teams.get">;
-  members: inferQueryOutput<"viewer.teams.get">["members"];
+  members: inferQueryOutput<"viewer.teams.getMembers">["members"];
```

Comment:
Since we are separating the "get" and "getMembers" endpoint we need to change the types here